pkg/cache/cache_test.go

```diff
@@ -22,40 +22,94 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	kcorev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	kmetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kcache "k8s.io/client-go/tools/cache"
 
 	"sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const testNamespaceOne = "test-namespace-1"
+const testNamespaceTwo = "test-namespace-2"
+
+var informerCache cache.Cache
+var stop chan struct{}
+var knownPod1 runtime.Object
+var knownPod2 runtime.Object
+var knownPod3 runtime.Object
```

Comment:
move these inside `Describe` (or reply here as to why they're not there).  It makes it clearer where they're used

Reply:
also, use a 

```go
var (

)
```
block